test/testGraphical.m

```diff
@@ -38,9 +38,9 @@
     %standalone mode!
     status = execute_type_stage(status, ipp);
 
-    if ~status.closeall && ~isempty(status.plotStage.fig_handle)
+    try
```

Comment:
Basically, there is a problem somewhere of the figure being already closed! This issue has already affected `testHeadless()`. In fact, there is a `try catch` there, together with if condistions ... and the code doesn't look very nice.